finetune/base_models/bert/roberta_encoder.py

```diff
@@ -44,34 +39,10 @@ def __init__(self, encoder_path=gpt2_encoder.ENCODER_PATH, vocab_path=gpt2_encod
                 )  # add 4 for the special tokens at beginning
                 index += 1
 
-
     def _convert_to_embed_idx(self, idx):
         return self.freqs[str(idx)]
-    
-
-    def _lazy_init(self, errors="replace"):
-        if self.initialized:
-            return
-
-        # Load encoder
-        with open(self.encoder_path, "r") as f:
-            self.encoder = json.load(f)
-            # shift all indices forward four places to make place for embeddings for start, pad, delim, clf
-            if self.offset != 0:
```

Comment:
I'm concerned about deleting this bit -- presumably this was needed?

Reply:
So, I moved this into gpt2 tokenizer and set the offset to 0 for that one.

Reply:
:+1: